pkgs/development/libraries/science/biology/ciftilib/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, boost
+, libxmlxx
+, pkg-config
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ciftilib";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Washington-University";
+    repo = "CiftiLib";
+    rev = "v${version}";
+    sha256 = "xc2dpMse4SozYEV/w3rXCrh1LKpTThq5nHB2y5uAD0A=";
+  };
+
+  nativeBuildInputs = [ boost cmake pkg-config ];
+  buildInputs = [ libxmlxx zlib ];
+
+  doCheck = true;
+  checkPhase = ''
+    runHook preCheck
+    ctest -E 'big|datatype-md5'
+    runHook postCheck
+  '';
```

Comment:
Using flags

``` cmake
  cmakeFlags = [
    "-DCMAKE_CTEST_ARGUMENTS=--exclude-regex;'big|datatype-md5'"
  ];
```
should also work, to avoid overriding the phase.

Reply:
Done.